### PR TITLE
RM-156363 Release 0.1.1

### PR DESCRIPTION
**Pull request by joecotton-wk**
## Purpose

Adding WK CI so that it builds correctly,

## Solution

High level overview of what was done. This is the roadmap for those who are going to CR.

## Semantic Versioning (check one)

- [ ] The following were changed in a non-backward compatible way and requires a major version bump:
  - _[link to the breaking change in the diff]_
- [ ] Something public was added or changed in backward compatible way, this requires a minor version bump
- [x] No public changes nor new features (backwards-compatible refactor or bug fix), so this can be included in a patch
      release

## How to QA

QA:

- [x] `go build ./...` runs successfully
- [x] Build passes
- [x] Skynet passes
- [x] Aviary passes
- [x] Rosie runs her checks

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
The items listed below may not capture all security relevant changes. Before providing a security review, be sure to review the entire PR for security impact.
<details>
<summary>(3) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>github.com/gogo/protobuf < 1.3.2</code> via <code>go.sum</code></li>
<li>1 vulns in <code>github.com/opencontainers/image-spec < 1.0.2</code> via <code>go.sum</code></li>
<li>6 vulns in <code>github.com/opencontainers/runc < 1.1.2</code> via <code>go.sum</code></li>
</details>
<details>
<summary>(1) Security relevant changes were detected</summary>
<li>Watched file <code>aviary.yaml</code> added</li>
</details>

## Action Items
* Request infosec review via Slack: #support-infosec
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=go-dexec">this Splunk dashboard</a> for more CVE details
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+contact: support-onecloud
+
+run:
+  on-pull-request: true
+  on-tag: true
+  when-branch-name-is:
+    - master
+
+scripts:
+  - echo This repository does not run any tests
```
Comment (austinarbor-wk) [inline skynet.yaml:16]: can we add the tests and vet here?

**Review by austinarbor-wk — CHANGES_REQUESTED** [review]
the repo builds fine, but we should add the tests and vet to the skynet run

File: skynet.yaml (hunk 1 above)
Comment (joecotton-wk) [reply skynet.yaml:16]: The tests are broken; they don't work on the master branch before these changes were applied. That's why I'm not running them.

As for vet - we're not currently running vet in any of these repos. The primary purpose of the migration was to move the repos over with the least amount of modification possible. Vet wasn't being run before, and so I haven't been adding it in. However, this is something we could add in the future.

Comment (austinarbor-wk) [reply skynet.yaml:16]: got it, ok

**Review by austinarbor-wk — APPROVED** [review]
LGTM

**Comment by joecotton-wk** [comment]
QA +1

**Comment by joecotton-wk** [comment]
@Workiva/release-management-p

**Comment by joecotton-wk** [comment]
Rosie please try again

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**